config/locales/it.yml

```diff
@@ -129,14 +161,22 @@ it:
       spree/tax_category:
         description: Descrizione
         name: Nome
+        is_default: Default
+        tax_code: Codice Tassa
       spree/tax_rate:
         amount: Tasso
         included_in_price: Incluso nel Prezzo
         show_rate_in_label: Mostra tasso nell'etichetta
+        name: Nome
       spree/taxon:
         name: Nome
         permalink: Permalink
         position: Posizione
+        description: Descrizione
+        icon: Icona
+        meta_description: Meta Description
+        meta_keywords: Meta Keywords
+        meta_title: Meta Title
```

Comment:
The above keys are not translated. Maybe they should? @alepore what do you think?